playground/frontend/lib/components/toggle_theme_button/toggle_theme_button.dart

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:flutter/material.dart';
+import 'package:playground/config/theme.dart';
+import 'package:playground/constants/sizes.dart';
+import 'package:provider/provider.dart';
+
+class ToggleThemeButton extends StatelessWidget {
+  const ToggleThemeButton({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Consumer<ThemeProvider>(builder: (context, theme, child) {
+      final text = theme.isDarkMode ? "Light Mode" : "Dark Mode";
```

Comment:
This comment not a blocker for the PR.

May we hoist the constants "Light mode" and "Dark mode" to the top of this file?

```dart
const kLightMode = "Light Mode";
const kDartMode = "Dark Mode";
```